modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsetsStructure.testcase

```diff
@@ -71,12 +71,63 @@ definition {
 	}
 
 	@description = "This is a test for LPS-123538. As Content Creator, I want to define reusable translations when creating WEM Structures with fieldset"
-	@ignore = "true"
 	@priority = "4"
 	test DefaultTitlePersistsUponFirstlyTranslatingForAnotherLocale {
 		property portal.acceptance = "false";
 
-		// TODO LPS-123538 DefaultTitlePersistsUponFirstlyTranslatingForAnotherLocale pending implementation
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "Structure Description Test",
+			structureName = "Structure Name Test");
+
+		Navigator.gotoSidebarTab(tabName = "Builder");
+
+		Navigator.gotoNavTab(navTab = "Fieldsets");
+
+		Click(locator1 = "FormViewBuilder#CREATE_NEW_FIELD_SETS");
+
+		DataEngine.addFields(fieldNameList = "Text");
+
+		Click.clickNoMouseOver(locator1 = "Button#LOCALIZATION_GENERIC_MODAL");
+
+		Click(
+			key_locale = "pt-BR",
+			locator1 = "Translation#LOCALIZATION_NAME_MODAL");
+
+		Type(
+			locator1 = "FormViewBuilder#FIELD_SET_NAME",
+			value1 = "New FieldSet PT");
+
+		Click.clickNoMouseOver(locator1 = "Button#LOCALIZATION_GENERIC_MODAL");
+
+		Click(
+			key_locale = "en-US",
+			locator1 = "Translation#LOCALIZATION_NAME_MODAL");
+
+		Type(
+			locator1 = "FormViewBuilder#FIELD_SET_NAME",
+			value1 = "New FieldSet");
+
+		Click(locator1 = "Modal#SAVE_BUTTON");
+
+		Alert.viewSuccessMessage();
+
+		DataEngine.gotoFieldsetItemEllipsis(fieldsetName = "New FieldSet");
+
+		MenuItem.click(menuItem = "Edit");
+
+		Click.clickNoMouseOver(locator1 = "Button#LOCALIZATION_GENERIC_MODAL");
+
+		Click(
+			key_locale = "pt-BR",
+			locator1 = "Translation#LOCALIZATION_NAME_MODAL");
+
+		AssertTextEquals(
```

Comment:
hey @gabrielargomes , here you will validate the default title fieldset name (english title).